### ISSUE TITLE
Unreadable text after resizing and ghost menus after closing them

### ISSUE THREAD

**Issue by TobiasSamsonsen** [desktop/desktop#16413]
### The problem

When I resize the Github Desktop window, the text becomes very blurry, making it almost impossible to read. This issue has been occurring for several months and affects both my laptop and desktop. Additionally, sometimes when I close a menu, a faint "ghost" of the menu remains on the screen.

### Release version

3.2.0

### Operating system

Windows 11

### Steps to reproduce the behavior

Text
1. Open Github Desktop on a screen with a resolution of 1920x1080 (might happen on other resolutions as well).
2. Move the small window around the desktop
3. Make the window full size
4. The text is now unclear

Menu
1. Open Github Desktop
2. Click on the menus at the top (file, edit, etc.)
3. After a few tries, the some of the menus should now show a ghost after them

### Log files

[2023-03-29.desktop.production.log](https://github.com/desktop/desktop/files/11102581/2023-03-29.desktop.production.log)

### Screenshots

![image](https://user-images.githubusercontent.com/116917837/228585945-52d76f18-3eba-4bc8-bb99-6851f3c5bc45.png)
![image](https://user-images.githubusercontent.com/116917837/228586296-6dc1c088-10b8-4d9f-b629-0a61b547dbe1.png)

### Additional context

Seems to be a problem with Github desktop as both my desktop and laptop has this problem. Surprisingly, I know two other people who haven't noticed it.

**Comment by tsvetilian-ty** [comment]
@TobiasSamsonsen thanks for the report. Your issue looks similar to https://github.com/desktop/desktop/issues/14190 could you try the solution suggested in https://github.com/desktop/desktop/issues/14190#issuecomment-1069594613?

**Comment by TobiasSamsonsen** [comment]
@tsvetilian-ty Thank you for the response. I checked Nvidia control panel and Antialiasing - FXAA was already set to off. Tried doing it all again, and found out that I probably had chosen the wrong .exe file. When changing the setting on the correct one it didn't work either. I figured there were some other anti-aliasing settings to check as well and ended with the conclusion that Multi-Frame Sampled AA (MFAA) was the problem. Changed that and everything was fixed.
